sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/exception/AmqpException.java

```diff
@@ -142,7 +142,7 @@ public ErrorCondition getErrorCondition() {
      *
      * @return The context for this exception.
      */
-    public ErrorContext getContext() {
+    public AmqpErrorContext getContext() {
```

Comment:
Same here `getAmqpErrorContext()` and in other places that have getters and setters for the renamed types.